server/internal/infrastructure/db/postgres/migration/20250627185230_update_intent_and_vtxo.up.sql

```diff
@@ -0,0 +1,42 @@
+DROP VIEW vtxo_virtual_tx_vw;
+DROP VIEW request_vtxo_vw;
+DROP VIEW vtxo_vw;
+DROP VIEW request_receiver_vw;
+DROP VIEW round_request_vw;
+
+ALTER TABLE tx_request ADD COLUMN proof TEXT NOT NULL;
+ALTER TABLE tx_request ADD COLUMN message TEXT NOT NULL;
```

Comment:
message could be JSONB type instead of TEXT (optimised for JSON + allow query by JSON field)

Reply:
we already encode the intent message for other reasons in the app svc, so it seemed the right call to store it as TEXT